addons/mail/static/src/messaging/component/partner_im_status_icon/partner_im_status_icon.scss

```diff
@@ -15,6 +15,12 @@
     transform: scale(1.5);
 }
 
+.o_PartnerMentionSuggestionDisplay {
+    transform: scale(1);
```

Comment:
`scale(1)` is default scale right? if so, remove rule

Reply:
task-2253613 if I'm not mistaken